### PR TITLE
[WIP] [PAUSED] team-configurable token expiry

### PR DESCRIPTION
**Pull request by ChrisPenner**
In progress;

- [x] Add `team_settings` table to Cassandra
- [x] Add `/i/teams/:tid/settings` endpoint for brig and friends to fetch settings for team
  - [x] GET
- [x] Add `/teams/:tid/settings` endpoint to allow team admins to get/set settings for team (via curl)
  - [x] GET
  - [x] PUT
- [x] Add team settings to Galley Env and Options
- [ ] Update all yaml configs to contain new required values

I plan to make a separate PR for:

- [ ] Use team settings from Galley when issuing tokens

### CODE REVIEW

**Comment by neongreen** [comment]
Ping me when there are "stable" commits that I can review already.

File: libs/galley-types/src/Galley/Types/Teams.hs [hunk 1]
```diff
@@ -199,6 +208,13 @@ data TeamUpdateData = TeamUpdateData
     , _iconKeyUpdate :: Maybe (Range 1 256 Text)
     } deriving (Eq, Show)
 
+data TeamSettings = TeamSettings
+    { _tsUserTokenTimeout     :: ZAuth.UserTokenTimeout
```
Comment (ChrisPenner) [inline libs/galley-types/src/Galley/Types/Teams.hs]: I just used the existing name; but I'm starting to think that `UserTokenExpiry` may be more descriptive ya?

**Comment by ChrisPenner** [comment]
This work is paused pending https://github.com/wearezeta/backend-issues/issues/773#issuecomment-474363384

**Comment by ChrisPenner** [comment]
Closing this for the forseeable future; we can find it if we need it.

**Pull request closed without merging**